### ISSUE TITLE
libyang.so.2.25.4 seems abnormal size

### ISSUE THREAD

**Issue by mx568549060** [CESNET/libyang#1965]
libyang-2.1.4  
after i cross compile,the libyang.so.2.25.4 seems abnormal size
`81M Jan  6 18:29 libyang.so.2.25.4`

In libyang-2.0.231，the so file ok.

**Comment by michalvasko** [comment]
Yes, that was a mistake on my part, current `devel` has already been fixed.
